extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -180,29 +170,21 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 						},
 						async (_progress, _token): Promise<void> => {
 							try {
+								let cr = this.schedulingParamsToEdit(this.saveArgs.coresRequest.w, this.saveArgs.coresRequest.c);
```

Comment:
The interface defines the properties such as coresRequest as optional but you're not checking for that here. Either update the interface to be non-optional or use null-coalescing

Reply:
Within the function if any of the properties are undefined, it will return undefined to the edit command.

Reply:
If coresRequest is undefined then it'll throw an error since (can't access property w on 'undefined') which I assume isn't what you want. Doing `this.saveArgs.coreRequest?.w` will pass in undefined and be handled there.